.github/workflows/microshed-ci.yml

```
@@ -17,14 +18,22 @@ env:
   LANG: en_US.UTF-8
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        java: [ 11 ]
+      fail-fast: false
+      max-parallel: 4
+    name: Build ${{ matrix.java }}, ${{ matrix.os }}
+
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Java
-      uses: joschi/setup-jdk@v2
+      uses: actions/setup-java@v3
       with:
         java-version: 11
```

Switch this to use the test matrix:
```suggestion
        java-version: ${{ matrix.java }}
```